test/Microsoft.Sbom.Api.Tests/Workflows/ManifestGenerationWorkflowTests.cs

```diff
@@ -453,7 +454,7 @@ public async Task ManifestGenerationWorkflowTests_SBOMDir_NotDefault_NotDeleted(
         using var manifestStream = new MemoryStream();
 
         fileSystemMock.Setup(f => f.DirectoryExists(It.IsAny<string>())).Returns(true);
-        var sbomConfig = new SbomConfig(fileSystemMock.Object)
+        ISbomConfig sbomConfig = new SbomConfig(fileSystemMock.Object)
```

Comment:
Changed 'var' to 'ISbomConfig` because `Mock.Setup` with an `out` parameter is very finicky about exactly matching the type